config/service_providers.yml

```diff
@@ -285,3 +285,28 @@ production:
     cert: 'identity_dashboard_cert'
     attribute_bundle:
       - email
+
+  # CBP Jobs
+  'urn:gov:dhs.cbp.jobs:openidconnect:cert':
+    redirect_uri: 'https://careers-cert.cbp.dhs.gov/hrm/app'
+    friendly_name: 'CBP Jobs'
+    agency: 'DHS'
+    logo: 'cbp.png'
+
+  'urn:gov:dhs.cbp.jobs:openidconnect:cert:app':
+    redirect_uri: 'gov.dhs.cbp.jobs.applicant.cert://result'
+    friendly_name: 'CBP Jobs'
+    agency: 'DHS'
+    logo: 'cbp.png'
+
+  'urn:gov:dhs.cbp.jobs:openidconnect:prod':
+    redirect_uri: 'https://careers.cbp.dhs.gov/hrm/app'
+    friendly_name: 'CBP Jobs'
+    agency: 'DHS'
+    logo: 'cbp.png'
+
+  'urn:gov:dhs.cbp.jobs:openidconnect:prod:app':
+    redirect_uri: 'gov.dhs.cbp.jobs.applicant://result'
```

Comment:
I'd defer to someone with more OIDC experience, but looks correct (see our entry for the [example iOS app](https://github.com/18F/identity-idp/blob/add-cbp-jobs-sps/config/service_providers.yml#L110)).